Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN yarn workspace client pack --filename app-hub-client.tgz
 RUN tar zxvf client/app-hub-client.tgz --directory server/
 RUN mv server/package/build server/static && rm -rf server/package
 
-FROM node:lts-slim
+FROM node:lts-buster-slim
```

Comment:
I think it is safe to use `slim` here for the production image, but we should pin the Node version:

`FROM node:14-buster-slim`